ably.d.ts

```diff
@@ -3081,7 +3081,6 @@ declare namespace Types {
      *
      * Creates a new {@link ChannelBase} or {@link RealtimeChannelBase} object, with the specified channel {@link DeriveOptions}
      * and {@link ChannelOptions}, or returns the existing channel object.
-     *
```

Comment:
Guessing this was staged by mistake? Having a line between description and params is consistent with the other jsdoc annotations

Reply:
Whoops, removed!